tools/bin/ecflow/processes/main/models/create_global_file.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+"""
+create_global_file.py
+usage: <python> <create_global_file.py> <configuration.cfg>
+
+This script creates a global file from the template with the correct
+ model start and end dates.
+"""
+import argparse
+from tonic.io import read_config
+from monitor import model_tools
+
+######### ----------------------------------------###########
+
+# read in configuration file
+parser = argparse.ArgumentParser(description='Reorder dimensions')
+parser.add_argument('config_file', metavar='config_file',
+                    type=argparse.FileType('r'), nargs=1,
+                    help='configuration file')
+args = parser.parse_args()
+config_dict = read_config(args.config_file[0].name)
```

Comment:
I don't think you want to use the `type=argparse.FileType('r')` argument here since it automatically opens the argument and then you have to backout the filename to use it in `read_config`.